README.md

```diff
@@ -91,17 +89,20 @@ app.post("/api/v1/orders", (req, res) => {
 });
 ```
 
-Running `napi split` will generate modular services based on these annotations. You'll have a `Users` service and an `Orders` service, each containing the respective endpoint.
+You can view more example in the [examples](/examples/)
+
+Running `napi split` with the following annotations will generate modular services based on these annotations. You'll have a `Users` service and an `Orders` service, each containing the respective endpoint.
 
 ## Using the UI
 
 The UI provides an interactive interface to:
 
 - Group and organize endpoints.
 - Preview the generated microservices.
-- Export refactored code for deployment.
 
-![NanoAPI UI Preview](/media/screenshots/app-ui.png)
+```
+napi split configure
```

Comment:
Will rewrite this whole section once we have dependency visualizer. So did not put much effort here. Just added the command. The image was broken